src/promptflow/promptflow/_sdk/_submitter/test_submitter.py

```diff
@@ -38,8 +39,27 @@
 
 
 class TestSubmitter:
-    def __init__(self, flow: Union[ProtectedFlow, EagerFlow], flow_context: FlowContext, client=None):
-        self.flow = flow
+    """
+    Submitter for testing flow/node.
+
+    A submitter will be bonded to a test run (including whether this is a node test or a flow test) after __init__,
+    and will be bonded to a specific executor proxy within an init context:
+    1) we will occupy some resources like a temporary folder to save flow with variant resolved, or an execution
+      service process if applicable;
+    2) output path will also be fixed within an init context;
+    usage:
+    test_submitter = TestSubmitter(...)
```

Comment:
add some comment that the with syntax is used to clean up excutor service resource?

Reply:
added